From b62e6a076d58adfa73d015e7c9091e527d5115a6 Mon Sep 17 00:00:00 2001
From: Terry Zhou <bjzhou@marvell.com>
Date: Mon, 27 Jun 2016 11:00:41 +0300
Subject: [PATCH 228/239] makefile: a3700: update the name of ddr configuration

we have new DDR4 board which need different settings with
same DDR frequency, update the configuration name from
ddr to ddr3

Change-Id: Ib82424e9baf5eeaf617aa4e93c8ff810936127e9
Signed-off-by: Terry Zhou <bjzhou@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30737
Reviewed-by: Hua Jing <jinghua@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
---
 Makefile               |   2 +-
 tools/wtp/buildtim.sh  |  12 +++---
 tools/wtp/ddr-600.txt  | 100 -------------------------------------------------
 tools/wtp/ddr-800.txt  | 100 -------------------------------------------------
 tools/wtp/ddr3-600.txt | 100 +++++++++++++++++++++++++++++++++++++++++++++++++
 tools/wtp/ddr3-800.txt | 100 +++++++++++++++++++++++++++++++++++++++++++++++++
 6 files changed, 207 insertions(+), 207 deletions(-)
 delete mode 100644 tools/wtp/ddr-600.txt
 delete mode 100644 tools/wtp/ddr-800.txt
 create mode 100644 tools/wtp/ddr3-600.txt
 create mode 100644 tools/wtp/ddr3-800.txt

diff --git a/Makefile b/Makefile
index 300644f..ee63d38 100644
--- a/Makefile
+++ b/Makefile
@@ -382,7 +382,7 @@ BUILD_UART		:= uart-images
 SRCPATH			:= $(dir $(BL33))
 
 CLOCKSPATH		:= $(DOIMAGEPATH)
-CLOCKSPRESET		?= CPU_800_DDR_800
+CLOCKSPRESET		?= CPU_800_DDR3_800
 
 BOOTDEV			?= SPINOR
 PARTNUM			?= 0
diff --git a/tools/wtp/buildtim.sh b/tools/wtp/buildtim.sh
index 76bd3c0..64af566 100755
--- a/tools/wtp/buildtim.sh
+++ b/tools/wtp/buildtim.sh
@@ -117,17 +117,17 @@ UART)
 esac
 
 case "$PRESET" in
-CPU_600_DDR_600)
+CPU_600_DDR3_600)
 	CLOCKSFILE=$CLOCKSPATH/clocks-600-600.$FILEEXT
-	DDRFILE=$CLOCKSPATH/ddr-600.$FILEEXT
+	DDRFILE=$CLOCKSPATH/ddr3-600.$FILEEXT
 	;;
-CPU_800_DDR_800)
+CPU_800_DDR3_800)
 	CLOCKSFILE=$CLOCKSPATH/clocks-800-800.$FILEEXT
-	DDRFILE=$CLOCKSPATH/ddr-800.$FILEEXT
+	DDRFILE=$CLOCKSPATH/ddr3-800.$FILEEXT
 	;;
-CPU_1000_DDR_800)
+CPU_1000_DDR3_800)
 	CLOCKSFILE=$CLOCKSPATH/clocks-1000-800.$FILEEXT
-	DDRFILE=$CLOCKSPATH/ddr-800.$FILEEXT
+	DDRFILE=$CLOCKSPATH/ddr3-800.$FILEEXT
 	;;
 *)
 	echo "Unsupported clock preset $PRESET!"
diff --git a/tools/wtp/ddr-600.txt b/tools/wtp/ddr-600.txt
deleted file mode 100644
index ebad38c..0000000
--- a/tools/wtp/ddr-600.txt
+++ /dev/null
@@ -1,100 +0,0 @@
-; Set DDR to 600MHz
-; Pre-init
-WRITE: 0xC0014008 0x00404500
-WRITE: 0xC0002000 0x00010000
-WRITE: 0xC0002004 0x00000000
-; DDR MAC init
-WRITE: 0xC0000340 0x0F0F0FEF
-WRITE: 0xC0000344 0x100000AA
-WRITE: 0xC0000310 0x00200000
-WRITE: 0xC0000304 0x00000000
-WRITE: 0xC0000308 0x00000000
-WRITE: 0xC0000200 0x000D0001
-WRITE: 0xC0000204 0x00000000
-WRITE: 0xC0000220 0x13020532
-WRITE: 0xC0000044 0x00030200
-WRITE: 0xC00002c0 0x00006000
-WRITE: 0xC00002c4 0x00100020
-WRITE: 0xC0000058 0x0000143F
-WRITE: 0xC0000048 0x00000001
-WRITE: 0xC0000180 0x00010200
-WRITE: 0xC0000050 0x000001FF
-WRITE: 0xC000004c 0x00000000
-WRITE: 0xC0000054 0x00000480
-WRITE: 0xC0000300 0x00000708
-WRITE: 0xC0000380 0x000001F5
-WRITE: 0xC0000384 0x000003E9
-WRITE: 0xC0000388 0x09600043
-WRITE: 0xC000038c 0x00000200
-WRITE: 0xC0000390 0x00400100
-WRITE: 0xC0000394 0x006B03CF
-WRITE: 0xC0000398 0x00720200
-WRITE: 0xC000039c 0x00120707
-WRITE: 0xC00003a0 0x00040511
-WRITE: 0xC00003a4 0x00000001
-WRITE: 0xC00003a8 0x00000C04
-WRITE: 0xC00003ac 0x15210919
-WRITE: 0xC00003b0 0x090b0609
-WRITE: 0xC00003b4 0x04000600
-WRITE: 0xC00003b8 0x00000600
-WRITE: 0xC00003bc 0x02020404
-WRITE: 0xC00003c0 0x00000000
-WRITE: 0xC00003c4 0x00000000
-WRITE: 0xC00003dc 0x00081239
-WRITE: 0xC00002c8 0x00000000
-WRITE: 0xC0000064 0x00000006
-WRITE: 0xC0000044 0x00030200
-; DDR PHY init
-WRITE: 0xC0001004 0x10077779
-WRITE: 0xC0001008 0x1FF00770
-WRITE: 0xC000100c 0x3F03FC77
-WRITE: 0xC0001010 0x00100118
-WRITE: 0xC0001028 0x00000000
-WRITE: 0xC0001030 0x03800000
-WRITE: 0xC0001034 0x00000000
-WRITE: 0xC0001040 0x00000400
-WRITE: 0xC00010c0 0x80000001
-WRITE: 0xC00010d0 0x00000000
-WRITE: 0xC00010e0 0x00011FF0
-WRITE: 0xC0001090 0x00000000
-WRITE: 0xC0001094 0x00000000
-WRITE: 0xC0001098 0x00000000
-WRITE: 0xC000109c 0x00000000
-WRITE: 0xC00010a0 0x00000000
-WRITE: 0xC00010a4 0x00000000
-WRITE: 0xC00010a8 0x00000000
-WRITE: 0xC00010ac 0x00000000
-WRITE: 0xC00010b0 0x00000000
-WRITE: 0xC0001000 0x00044041
-WRITE: 0xC0001014 0x00080200
-WRITE: 0xC0001038 0x00000002
-WRITE: 0xC000103c 0x00000010
-WRITE: 0xC0001180 0x0000020A
-WRITE: 0xC0001184 0x0000020A
-WRITE: 0xC0001188 0x0000020A
-WRITE: 0xC000118c 0x0000020A
-WRITE: 0xC0001190 0x0000020A
-WRITE: 0xC0001194 0x0000020A
-WRITE: 0xC0001198 0x0000020A
-WRITE: 0xC000119c 0x0000020A
-WRITE: 0xC00011a0 0x0000020A
-WRITE: 0xC0001050 0x08080000
-WRITE: 0xC0001054 0x08080000
-WRITE: 0xC0001058 0x08080000
-WRITE: 0xC000105c 0x08080000
-WRITE: 0xC0001060 0x08080000
-WRITE: 0xC0001064 0x08080000
-WRITE: 0xC0001068 0x08080000
-WRITE: 0xC000106c 0x08080000
-WRITE: 0xC0001070 0x08080000
-WRITE: 0xC0001074 0x20200000
-WRITE: 0xC0001020 0x20000000
-WRITE: 0xC0001020 0x40000000
-WRITE: 0xC0001020 0x80000000
-; Trigger DDR init for channel 0
-WRITE: 0xC0000020 0x11000001
-WAIT_FOR_BIT_SET: 0xC0000008 0x00000001 0x00001000
-; AVS enable
-WRITE: 0xC0011500 0x98A2FFFF	;# Set VDD to 1150mV and soft reset AVS
-WRITE: 0xC0011500 0x18A2FFFF	;# release AVS reset
-WRITE: 0xC0011500 0x58A2FFFF	;# Enable AVS
diff --git a/tools/wtp/ddr-800.txt b/tools/wtp/ddr-800.txt
deleted file mode 100644
index c3225d2..0000000
--- a/tools/wtp/ddr-800.txt
+++ /dev/null
@@ -1,100 +0,0 @@
-; Set DDR to 800MHz
-; Pre-init
-WRITE: 0xC0014008 0x00404500
-WRITE: 0xC0002000 0x00010000
-WRITE: 0xC0002004 0x00000000
-; DDR MAC init
-WRITE: 0xC0000340 0x0F0F0FEF
-WRITE: 0xC0000344 0x100000AA
-WRITE: 0xC0000310 0x00200000
-WRITE: 0xC0000304 0x00000000
-WRITE: 0xC0000308 0x00000000
-WRITE: 0xC0000200 0x000D0001
-WRITE: 0xC0000204 0x00000000
-WRITE: 0xC0000220 0x13020532
-WRITE: 0xC0000044 0x00030200
-WRITE: 0xC00002c0 0x00006000
-WRITE: 0xC00002c4 0x00100020
-WRITE: 0xC0000058 0x0000143F
-WRITE: 0xC0000048 0x00000001
-WRITE: 0xC0000180 0x00010200
-WRITE: 0xC0000050 0x000001FF
-WRITE: 0xC000004c 0x00000000
-WRITE: 0xC0000054 0x00000480
-WRITE: 0xC0000300 0x0000080B
-WRITE: 0xC0000380 0x0007A120
-WRITE: 0xC0000384 0x00030D40
-WRITE: 0xC0000388 0x0960006B
-WRITE: 0xC000038c 0x00000200
-WRITE: 0xC0000390 0x00400100
-WRITE: 0xC0000394 0x00F003CF
-WRITE: 0xC0000398 0x00F80200
-WRITE: 0xC000039c 0x00000808
-WRITE: 0xC00003a0 0x00040614
-WRITE: 0xC00003a4 0x00000001
-WRITE: 0xC00003a8 0x00000C04
-WRITE: 0xC00003ac 0x202A0C1F
-WRITE: 0xC00003b0 0x0C0C060C
-WRITE: 0xC00003b4 0x04000600
-WRITE: 0xC00003b8 0x00000800
-WRITE: 0xC00003bc 0x02020404
-WRITE: 0xC00003c0 0x00000000
-WRITE: 0xC00003c4 0x00000000
-WRITE: 0xC00003dc 0x00081239
-WRITE: 0xC00002c8 0x00000000
-WRITE: 0xC0000064 0x00000006
-WRITE: 0xC0000044 0x00030200
-; DDR PHY init
-WRITE: 0xC0001004 0x10077779
-WRITE: 0xC0001008 0x1FF00770
-WRITE: 0xC000100c 0x3F03FC77
-WRITE: 0xC0001010 0x00100118
-WRITE: 0xC0001028 0x00000000
-WRITE: 0xC0001030 0x03800000
-WRITE: 0xC0001034 0x00000000
-WRITE: 0xC0001040 0x00000400
-WRITE: 0xC00010c0 0x80000001
-WRITE: 0xC00010d0 0x00000000
-WRITE: 0xC00010e0 0x00011FF0
-WRITE: 0xC0001090 0x00000000
-WRITE: 0xC0001094 0x00000000
-WRITE: 0xC0001098 0x00000000
-WRITE: 0xC000109c 0x00000000
-WRITE: 0xC00010a0 0x00000000
-WRITE: 0xC00010a4 0x00000000
-WRITE: 0xC00010a8 0x00000000
-WRITE: 0xC00010ac 0x00000000
-WRITE: 0xC00010b0 0x00000000
-WRITE: 0xC0001000 0x00044041
-WRITE: 0xC0001014 0x00080200
-WRITE: 0xC0001038 0x00000002
-WRITE: 0xC000103c 0x00000010
-WRITE: 0xC0001180 0x0000020A
-WRITE: 0xC0001184 0x0000020A
-WRITE: 0xC0001188 0x0000020A
-WRITE: 0xC000118c 0x0000020A
-WRITE: 0xC0001190 0x0000020A
-WRITE: 0xC0001194 0x0000020A
-WRITE: 0xC0001198 0x0000020A
-WRITE: 0xC000119c 0x0000020A
-WRITE: 0xC00011a0 0x0000020A
-WRITE: 0xC0001050 0x20200000
-WRITE: 0xC0001054 0x20200000
-WRITE: 0xC0001074 0x20200000
-WRITE: 0xC0001058 0x08080000
-WRITE: 0xC000105c 0x08080000
-WRITE: 0xC0001060 0x08080000
-WRITE: 0xC0001064 0x08080000
-WRITE: 0xC0001068 0x08080000
-WRITE: 0xC000106c 0x08080000
-WRITE: 0xC0001070 0x08080000
-WRITE: 0xC0001020 0x20000000
-WRITE: 0xC0001020 0x40000000
-WRITE: 0xC0001020 0x80000000
-; Trigger DDR init for channel 0
-WRITE: 0xC0000020 0x11000001
-WAIT_FOR_BIT_SET: 0xC0000008 0x00000001 0x00001000
-; AVS enable
-WRITE: 0xC0011500 0x98A2FFFF	;# Set VDD to 1150mV and soft reset AVS
-WRITE: 0xC0011500 0x18A2FFFF	;# release AVS reset
-WRITE: 0xC0011500 0x58A2FFFF	;# Enable AVS
diff --git a/tools/wtp/ddr3-600.txt b/tools/wtp/ddr3-600.txt
new file mode 100644
index 0000000..b79d43c
--- /dev/null
+++ b/tools/wtp/ddr3-600.txt
@@ -0,0 +1,100 @@
+; Set DDR3 to 600MHz
+; Pre-init
+WRITE: 0xC0014008 0x00404500
+WRITE: 0xC0002000 0x00010000
+WRITE: 0xC0002004 0x00000000
+; DDR MAC init
+WRITE: 0xC0000340 0x0F0F0FEF
+WRITE: 0xC0000344 0x100000AA
+WRITE: 0xC0000310 0x00200000
+WRITE: 0xC0000304 0x00000000
+WRITE: 0xC0000308 0x00000000
+WRITE: 0xC0000200 0x000D0001
+WRITE: 0xC0000204 0x00000000
+WRITE: 0xC0000220 0x13020532
+WRITE: 0xC0000044 0x00030200
+WRITE: 0xC00002c0 0x00006000
+WRITE: 0xC00002c4 0x00100020
+WRITE: 0xC0000058 0x0000143F
+WRITE: 0xC0000048 0x00000001
+WRITE: 0xC0000180 0x00010200
+WRITE: 0xC0000050 0x000001FF
+WRITE: 0xC000004c 0x00000000
+WRITE: 0xC0000054 0x00000480
+WRITE: 0xC0000300 0x00000708
+WRITE: 0xC0000380 0x000001F5
+WRITE: 0xC0000384 0x000003E9
+WRITE: 0xC0000388 0x09600043
+WRITE: 0xC000038c 0x00000200
+WRITE: 0xC0000390 0x00400100
+WRITE: 0xC0000394 0x006B03CF
+WRITE: 0xC0000398 0x00720200
+WRITE: 0xC000039c 0x00120707
+WRITE: 0xC00003a0 0x00040511
+WRITE: 0xC00003a4 0x00000001
+WRITE: 0xC00003a8 0x00000C04
+WRITE: 0xC00003ac 0x15210919
+WRITE: 0xC00003b0 0x090b0609
+WRITE: 0xC00003b4 0x04000600
+WRITE: 0xC00003b8 0x00000600
+WRITE: 0xC00003bc 0x02020404
+WRITE: 0xC00003c0 0x00000000
+WRITE: 0xC00003c4 0x00000000
+WRITE: 0xC00003dc 0x00081239
+WRITE: 0xC00002c8 0x00000000
+WRITE: 0xC0000064 0x00000006
+WRITE: 0xC0000044 0x00030200
+; DDR PHY init
+WRITE: 0xC0001004 0x10077779
+WRITE: 0xC0001008 0x1FF00770
+WRITE: 0xC000100c 0x3F03FC77
+WRITE: 0xC0001010 0x00100118
+WRITE: 0xC0001028 0x00000000
+WRITE: 0xC0001030 0x03800000
+WRITE: 0xC0001034 0x00000000
+WRITE: 0xC0001040 0x00000400
+WRITE: 0xC00010c0 0x80000001
+WRITE: 0xC00010d0 0x00000000
+WRITE: 0xC00010e0 0x00011FF0
+WRITE: 0xC0001090 0x00000000
+WRITE: 0xC0001094 0x00000000
+WRITE: 0xC0001098 0x00000000
+WRITE: 0xC000109c 0x00000000
+WRITE: 0xC00010a0 0x00000000
+WRITE: 0xC00010a4 0x00000000
+WRITE: 0xC00010a8 0x00000000
+WRITE: 0xC00010ac 0x00000000
+WRITE: 0xC00010b0 0x00000000
+WRITE: 0xC0001000 0x00044041
+WRITE: 0xC0001014 0x00080200
+WRITE: 0xC0001038 0x00000002
+WRITE: 0xC000103c 0x00000010
+WRITE: 0xC0001180 0x0000020A
+WRITE: 0xC0001184 0x0000020A
+WRITE: 0xC0001188 0x0000020A
+WRITE: 0xC000118c 0x0000020A
+WRITE: 0xC0001190 0x0000020A
+WRITE: 0xC0001194 0x0000020A
+WRITE: 0xC0001198 0x0000020A
+WRITE: 0xC000119c 0x0000020A
+WRITE: 0xC00011a0 0x0000020A
+WRITE: 0xC0001050 0x08080000
+WRITE: 0xC0001054 0x08080000
+WRITE: 0xC0001058 0x08080000
+WRITE: 0xC000105c 0x08080000
+WRITE: 0xC0001060 0x08080000
+WRITE: 0xC0001064 0x08080000
+WRITE: 0xC0001068 0x08080000
+WRITE: 0xC000106c 0x08080000
+WRITE: 0xC0001070 0x08080000
+WRITE: 0xC0001074 0x20200000
+WRITE: 0xC0001020 0x20000000
+WRITE: 0xC0001020 0x40000000
+WRITE: 0xC0001020 0x80000000
+; Trigger DDR init for channel 0
+WRITE: 0xC0000020 0x11000001
+WAIT_FOR_BIT_SET: 0xC0000008 0x00000001 0x00001000
+; AVS enable
+WRITE: 0xC0011500 0x98A2FFFF	;# Set VDD to 1150mV and soft reset AVS
+WRITE: 0xC0011500 0x18A2FFFF	;# release AVS reset
+WRITE: 0xC0011500 0x58A2FFFF	;# Enable AVS
diff --git a/tools/wtp/ddr3-800.txt b/tools/wtp/ddr3-800.txt
new file mode 100644
index 0000000..59a9405
--- /dev/null
+++ b/tools/wtp/ddr3-800.txt
@@ -0,0 +1,100 @@
+; Set DDR3 to 800MHz
+; Pre-init
+WRITE: 0xC0014008 0x00404500
+WRITE: 0xC0002000 0x00010000
+WRITE: 0xC0002004 0x00000000
+; DDR MAC init
+WRITE: 0xC0000340 0x0F0F0FEF
+WRITE: 0xC0000344 0x100000AA
+WRITE: 0xC0000310 0x00200000
+WRITE: 0xC0000304 0x00000000
+WRITE: 0xC0000308 0x00000000
+WRITE: 0xC0000200 0x000D0001
+WRITE: 0xC0000204 0x00000000
+WRITE: 0xC0000220 0x13020532
+WRITE: 0xC0000044 0x00030200
+WRITE: 0xC00002c0 0x00006000
+WRITE: 0xC00002c4 0x00100020
+WRITE: 0xC0000058 0x0000143F
+WRITE: 0xC0000048 0x00000001
+WRITE: 0xC0000180 0x00010200
+WRITE: 0xC0000050 0x000001FF
+WRITE: 0xC000004c 0x00000000
+WRITE: 0xC0000054 0x00000480
+WRITE: 0xC0000300 0x0000080B
+WRITE: 0xC0000380 0x0007A120
+WRITE: 0xC0000384 0x00030D40
+WRITE: 0xC0000388 0x0960006B
+WRITE: 0xC000038c 0x00000200
+WRITE: 0xC0000390 0x00400100
+WRITE: 0xC0000394 0x00F003CF
+WRITE: 0xC0000398 0x00F80200
+WRITE: 0xC000039c 0x00000808
+WRITE: 0xC00003a0 0x00040614
+WRITE: 0xC00003a4 0x00000001
+WRITE: 0xC00003a8 0x00000C04
+WRITE: 0xC00003ac 0x202A0C1F
+WRITE: 0xC00003b0 0x0C0C060C
+WRITE: 0xC00003b4 0x04000600
+WRITE: 0xC00003b8 0x00000800
+WRITE: 0xC00003bc 0x02020404
+WRITE: 0xC00003c0 0x00000000
+WRITE: 0xC00003c4 0x00000000
+WRITE: 0xC00003dc 0x00081239
+WRITE: 0xC00002c8 0x00000000
+WRITE: 0xC0000064 0x00000006
+WRITE: 0xC0000044 0x00030200
+; DDR PHY init
+WRITE: 0xC0001004 0x10077779
+WRITE: 0xC0001008 0x1FF00770
+WRITE: 0xC000100c 0x3F03FC77
+WRITE: 0xC0001010 0x00100118
+WRITE: 0xC0001028 0x00000000
+WRITE: 0xC0001030 0x03800000
+WRITE: 0xC0001034 0x00000000
+WRITE: 0xC0001040 0x00000400
+WRITE: 0xC00010c0 0x80000001
+WRITE: 0xC00010d0 0x00000000
+WRITE: 0xC00010e0 0x00011FF0
+WRITE: 0xC0001090 0x00000000
+WRITE: 0xC0001094 0x00000000
+WRITE: 0xC0001098 0x00000000
+WRITE: 0xC000109c 0x00000000
+WRITE: 0xC00010a0 0x00000000
+WRITE: 0xC00010a4 0x00000000
+WRITE: 0xC00010a8 0x00000000
+WRITE: 0xC00010ac 0x00000000
+WRITE: 0xC00010b0 0x00000000
+WRITE: 0xC0001000 0x00044041
+WRITE: 0xC0001014 0x00080200
+WRITE: 0xC0001038 0x00000002
+WRITE: 0xC000103c 0x00000010
+WRITE: 0xC0001180 0x0000020A
+WRITE: 0xC0001184 0x0000020A
+WRITE: 0xC0001188 0x0000020A
+WRITE: 0xC000118c 0x0000020A
+WRITE: 0xC0001190 0x0000020A
+WRITE: 0xC0001194 0x0000020A
+WRITE: 0xC0001198 0x0000020A
+WRITE: 0xC000119c 0x0000020A
+WRITE: 0xC00011a0 0x0000020A
+WRITE: 0xC0001050 0x20200000
+WRITE: 0xC0001054 0x20200000
+WRITE: 0xC0001074 0x20200000
+WRITE: 0xC0001058 0x08080000
+WRITE: 0xC000105c 0x08080000
+WRITE: 0xC0001060 0x08080000
+WRITE: 0xC0001064 0x08080000
+WRITE: 0xC0001068 0x08080000
+WRITE: 0xC000106c 0x08080000
+WRITE: 0xC0001070 0x08080000
+WRITE: 0xC0001020 0x20000000
+WRITE: 0xC0001020 0x40000000
+WRITE: 0xC0001020 0x80000000
+; Trigger DDR init for channel 0
+WRITE: 0xC0000020 0x11000001
+WAIT_FOR_BIT_SET: 0xC0000008 0x00000001 0x00001000
+; AVS enable
+WRITE: 0xC0011500 0x98A2FFFF	;# Set VDD to 1150mV and soft reset AVS
+WRITE: 0xC0011500 0x18A2FFFF	;# release AVS reset
+WRITE: 0xC0011500 0x58A2FFFF	;# Enable AVS
-- 
1.9.1

